telegrambots-abilities/src/test/java/org/telegram/abilitybots/api/bot/DefaultBot.java

```diff
@@ -79,6 +85,24 @@ public Reply channelPostReply() {
     );
   }
 
+  public ReplyCollection createReplyCollection() {
+    List<Reply> replyList = new ArrayList<>();
+    replyList.add(
+            Reply.of(
+                    upd -> silent.send("first reply answer", upd.getMessage().getChatId()),
+                    update -> update.getMessage().getText().equalsIgnoreCase(FIRST_REPLY_KEY_MESSAGE)
+            )
+    );
+    replyList.add(
+            Reply.of(
+                    upd -> silent.send("second reply answer", upd.getMessage().getChatId()),
+                    update -> update.getMessage().getText().equalsIgnoreCase(SECOND_REPLY_KEY_MESSAGE)
+            )
+
+    );
+    return new ReplyCollection(replyList);
```

Comment:
It would be a more conformant model if this looks like:

```return ReplyCollection.of( <R1>, <R2>, ... )```

That way, the internal collection used is internalized and the developer doesn't need to create the list himself.